corehq/form_processor/reprocess.py

```diff
@@ -48,7 +48,14 @@ def reprocess_unfinished_stub(stub, save=True):
         return
 
     assert form.is_normal
-    result = _reprocess_form(form, save)
+    try:
+        result = _reprocess_form(form, save)
+    except:
+        if save:
+            stub.queued = False
+            stub.save()
```

Comment:
Isn't that basically the same as what we had before where `timeout = 24 hours`.

Just to confirm my understanding, this should be `> (max time to process a batch)`. ie. if we're enqueuing 100 times at a time then this timeout should be `> 100 * (max time to process individual item)`.

Calculating that for this queue is a bit tricky since a case with only a few forms will process quickly but a case with 1000 forms will take a long time to process.